exporting/prometheus/README.md

```diff
@@ -13,7 +20,7 @@ are starting at a fresh ubuntu shell (whether you'd like to follow along in a VM
 
 ### Installing Netdata
 
-There are number of ways to install Netdata according to [Installation](../../packaging/installer/). The suggested way
+There are number of ways to install Netdata according to [Installation](/packaging/installer/README.md). The suggested way
```

Comment:
Should we not use `../packaging/installer/README.md` here?

Reply:
Joel changed the way we should write links in #8638.